src/js/langData.js

```diff
@@ -4,6 +4,7 @@ export const arrLang = {
     'menu-production': 'Produkcja',
     'menu-about': 'O firmie',
     'menu-contact': 'Kontakt',
+    'menu-rodo': 'Rodo',
```

Comment:
- [ ] Please change it to uppercase

Reply:
Ok, but I thought we were trying to avoid uppercases

Reply:
Yes, but in this case it is the name